CHANGELOG.md

```diff
@@ -1,124 +1,150 @@
 # Change Log
 
+## 0.15.0 - 2021-07-<???>
+
+### Added
+* Adapt bind_socket to make it usable with multiple processes (#1009) 21/06/21
+* Add existence check to the reload directory(ies) (#1089) 21/06/21
+* Add missing trace log for websocket protocols (#1083) 19/06/21
+* Add option to set websocket ping interval and timeout (#1048) 09/06/21
+* Docs: Add uvloop and httptools related info (#1071) 09/06/21
```

Comment:
I think it's better if we don't add docs additions from the changelog. It's also more aligned with [httpx](https://www.python-httpx.org/contributing/#releasing).
```suggestion
```